Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+- Met with Nottingham to support them as users of the DSH codebase
+- The team worked together on the content for DSH activities on RSEcon  as well as the UK TRE community satellite event #46 . 
+- The team visited the Bennett Institute for a show and tell about DSH and OpenSafely https://github.com/alan-turing-institute/tps-project-management/issues/157
+  - No immediate collaboration but agreed to be involved in the specification and TRE community
+- For discussion: this story needs to be redefined to include external engagement or a new one created
+
+## TRESA
+Over the year TRESA have increased its autonomy from the DSH research project, in terms of work ownership and management.
+Therefore TRESA stories have not been indepently updated and it is more comprenhensive to update on the service area as a whole.
+
+This warrants updating and reviewing the stories we keep under the DSH roadmap, focusing on communicating with the service area rahter than planning or prioritising for them.
+
+### Updates
+A new team of have been onboarded, who have focused on learning the ropes while simultaneously supporting the December DSG.
+With some support from the DSH team they were able to do so.
+
+We had a detailed[ handover meeting](https://hackmd.io/kh6siuZcTdCxcfYryAvypw) to go over each process, the idea is that moving forward TRESA directly owns processes and their improvement.
+
+We have also identified areas of work for TRESA to fully establish itself as a service area, these need to be elaborated into a full proposal with sprints and stories but main areas are:
+- Data protection
+- Cost and recovery model
+- Acredditation: ISO 27001 and DSPT renewal
+- Client projects management
+
+
+## Meta
+Work and stories that do not belong directly in any pillars but are necessary for all
+
+### Project strategy and ways of work
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/43
+
+#### Goal
+The aim is to develop a project strategy and revise best ways of work to achieve it
+
+Through several strategy sessions we will:
+- Define our north star (vision & mision)
+- Establish the project pilars or areas, defining what success looks like for each
+- Prioritise the measure of success, which are essential to consider the project succesful
+- Identify work required to achieve success
+- Allocate work by team
+- Produce an initial roadmap
+- Evaluate required effort for the work against team capacity
+- Develop and agree new ways of work, including meeting structure and use of project's repositories and projects
+
+#### Progress
+
+##### Strategy
+
+Through several team wide sessions we jointly produced a [project strategy](https://thealanturininstitute.sharepoint.com/:p:/s/SafeHaven/Ebrp4Iyc9M1NpPTgpgHdj5kB7HPvH-2gM0oNd97jJu6oxw?e=eN0ZFw)https://thealanturininstitute.sharepoint.com/:p:/s/SafeHaven/Ebrp4Iyc9M1NpPTgpgHdj5kB7HPvH-2gM0oNd97jJu6oxw?e=eN0ZFw a long, medium and short term levels.
```

Comment:
This is great and I'm super proud of it - I think it needs to be integrated into the GitHub repository to make it more accessible..... could that be a project management task for the next quarter?

Reply:
Pending commit until I capture that action as a to do